model/src/test/test_model_hospitalizations.py

```diff
@@ -367,7 +367,7 @@ def test_model_hosp_with_obs_model_weekday_phosp():
         infection_to_admission_interval=inf_hosp,
         infections_varname="infections",
         weekday_effect_dist=weekday,
-        p_report_dist=p_hosp,
+        hosp_report_prob_dist=p_hosp,
```

Comment:
rename `p_hosp` to `p_report` in line 363 and here. Also, shouldn't it be `p_report = DeterministicVariable((<some scalar>,))`, not `p_report = DeterministicVariable((weekday,))`?

Reply:
If we're testing a vector-valued `p_report`, then for robustness shouldn't it be a different vector from `weekday`?

Reply:
Sure, I'll make it separate before merging. Thanks!